ci-operator/jobs/openshift/gcp-pd-csi-driver-operator/openshift-gcp-pd-csi-driver-operator-main-periodics.yaml

```diff
@@ -4,23 +4,15 @@ periodics:
   cron: '@weekly'
   decorate: true
   extra_refs:
-  - base_ref: master
+  - base_ref: main
     org: openshift
     repo: gcp-pd-csi-driver-operator
   labels:
     ci-operator.openshift.io/cloud: gcp
     ci-operator.openshift.io/cloud-cluster-profile: gcp
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-gcp-pd-csi-driver-operator-master-periodic-e2e-gcp-csi-c3
-  reporter_config:
```

Comment:
Do not delete the reporter_config part, I think you need to return this manually after regeneration

Reply:
> Do not delete the reporter_config part, I think you need to return this manually after regeneration

@rhrmo good catch and thanks! my automation doesn't account for that kind of change.

Reply:
@DennisPerique reporter_config part got deleted again

Reply:
ah, I forgot that auto-rebase doesn't work in this case -- manually fixed. Thx for catching.